src/mjolnir/graphvalidator.cc

```diff
@@ -134,6 +134,10 @@ uint32_t GetOpposingEdgeIndex(const GraphId& startnode,
         continue;
       }
 
+      if (edge.length() > kMaxEdgeLength) {
+        throw std::runtime_error("Edge Length too Long");
+      }
```

Comment:
this is probably already taken of: https://github.com/gis-ops/valhalla/blob/6dac5913835663881178537d53e3385cfdf6d695/src/baldr/directededge.cc#L117-L123